include/revng/EarlyFunctionAnalysis/efa-schema.yml

```diff
@@ -37,14 +38,17 @@ definitions:
     fields:
       - name: ID
         type: BasicBlockID
+        default: ":Invalid"
```

Comment:
Can we omit this?
What happens if we do?

Reply:
Ah, wait! No! That's what would happen c++-side. For python and typescript, `BasicBlockID` is just a string - so it will stay as is (`""`).

Reply:
Is it just a string? We have `MetaAddress` in Python/TypeScript, don't we have `BasicBlockID`?

Reply:
I didn't look to deeply, but inside _TTG_ they are all threated like strings.

They might have more explicit representation later, when running _generated_ code. But this part focuses on generation-time. And at generation-time those are strings.